campus.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: UAVs on Campus
 ---
 ## Policy
-Flying a drone over campus requires that pilots abide by <a href="https://www.faa.gov/uas/faqs/" target="_blank">FAA guidelines</a> and UC Berkeley drone policy. For more information about drone policy within the UC system, refer to the UC Office of the President's <a href="https://www.ucop.edu/enterprise-risk-management/resources/centers-of-excellence/unmanned-aircraft-systems-safety.html" target="_blank">UAS Safety site</a>. In short, it is against Berkeley policy to fly a drone weighing more than 250 grams over campus property without written permission from UCOP UAS safety, UC Berkeley risk managment, and UCPD (this process requires the pilot to hold an <a href="https://www.faa.gov/news/fact_sheets/news_story.cfm?newsId=20516" target="_blank">FAA Part 107 certificate</a>). To view UC Berkeley's complete drone policy click <a href="https://campuspol.berkeley.edu/policies/drone(campus).pdf" target="_blank">here</a>.
+Flying a drone over campus requires that pilots abide by <a href="https://www.faa.gov/uas/faqs/" target="_blank">FAA guidelines</a> and UC Berkeley drone policy. For more information about drone policy within the UC system, refer to the UC Office of the President's <a href="https://www.ucop.edu/enterprise-risk-management/resources/centers-of-excellence/unmanned-aircraft-systems-safety.html" target="_blank">UAS Safety site</a>. In short, it is against Berkeley policy to fly a drone over campus property without written permission from UCOP UAS safety, UC Berkeley risk managment, and UCPD (this process requires a pilot of a drone weighing more than 250 grams to hold an <a href="https://www.faa.gov/news/fact_sheets/news_story.cfm?newsId=20516" target="_blank">FAA Part 107 certificate</a>). To view UC Berkeley's complete drone policy click <a href="https://campuspol.berkeley.edu/policies/drone(campus).pdf" target="_blank">here</a>.
```

Comment:
Thanks for working on this. A few edits for your review.  
1. Clarify policy
    > ... it is against Berkeley policy ...

    I would clarify this to be UC Berkeley, since it maybe interpreted as City of Berkeley. (I understand this was from before)  
2. Update the broken UCOP link
https://www.ucop.edu/safety-and-loss-prevention/environmental/program-resources/unmanned-aircraft-systems-safety.html
3. Delete reference to Part 107
> (this process requires a pilot of a drone weighing more than 250 grams to hold an <a href="https://www.faa.gov/news/fact_sheets/news_story.cfm?newsId=20516" target="_blank">FAA Part 107 certificate</a>)
We are able to fly under FAA Part 107 regulation, the pilot may not necessarily need to hold the certificate, unless they are flying commercially. So this line is just confusing.